src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const [itemFreq, setItemFreq] = useState(7);
+  const [formValidationVisual, setFormValidationVisual] = useState(false);
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(capitalize(event.target.value));
```

Comment:
To add to this, if you wanted to capitalize each word in the string instead of just the first word you could do:

```suggestion
    const groceryText = event.target.value.split(' ').map(capitalize).join(' ');
    setGroceryItem(groceryText);
```